clients/service-center-client/src/main/java/org/apache/servicecomb/service/center/client/ServiceCenterClient.java

```diff
@@ -422,4 +425,90 @@ public void sendHeartBeats(HeartbeatsRequest heartbeatsRequest) {
           "heartbeats fails ", e);
     }
   }
+
+  /**
+   * Get schemas list of service
+   *
+   * @param serviceId
+   * @return
+   * @throws OperationException
+   */
+  public List<GetSchemaResponse> getServiceSchemasList(String serviceId) {

```

Comment:
```GetSchemaResponse``` is used as an ```request``` in ```updateServiceSchemaContext```, it's better to change to a new name. e.g. ```SchemaInfo```

Reply:
Thanks for your suggestion, it has been modified.